src/tiledimage.js

```diff
@@ -778,6 +783,21 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
+     * @returns {Boolean} whether the tiledImage can load its tiles even when it has zero opacity.
+     */
+    getPreload: function() {
+        return this._preload;
+    },
+
+    /**
+     * Set true to load even when hidden. Set false to block loading when hidden.
+     */
+    setPreload: function(preload) {
+        this._preload = new Boolean(preload).valueOf();
```

Comment:
@thejohnhoffer Glad to see you back on the case! :)

We don't support boolean objects anywhere else in the OpenSeadragon code base, so I'm happy to stick with just the boolean primitive. In my review comment I was just confirming that this was fine with you as well. I don't mind having the boolean object support if you need it for your purposes. In that case you'd just need to prefix that line with:

```
// eslint-disable-next-line no-new-wrappers
```

Since we're converting it back to a primitive straight away, I don't see any problem with having it there; boolean objects have existed in all browsers since ie6. 

Anyway, I'm fine either way.